dom/test/browser/src/events.ts

```diff
@@ -873,4 +873,70 @@ describe('DOMSource.events()', function () {
     run();
   });
 
+  it('should allow preventing default event behavior with function', function (done) {
```

Comment:
Would be good to have tests also for the *opposite* case. If the implementation is `if (true) ev.preventDefault()` then these tests would pass.